hypertrace-ingester/build.gradle.kts

```diff
@@ -25,12 +29,12 @@ dependencies {
   implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.9")
   implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.8")
   implementation("org.hypertrace.core.datamodel:data-model:0.1.9")
-
-  implementation("org.hypertrace.core.spannormalizer:span-normalizer")
-  implementation("org.hypertrace.core.rawspansgrouper:raw-spans-grouper")
-  implementation("org.hypertrace.traceenricher:hypertrace-trace-enricher")
-  implementation("org.hypertrace.viewgenerator:hypertrace-view-generator")
   implementation("org.hypertrace.core.viewgenerator:view-generator-framework:0.1.14")
+
+  implementation(project(":span-normalizer:span-normalizer"))
```

Comment:
this is broke for me locally. I will push it to the other branch and we can fix it there.